lib/api/src/store.rs

```diff
@@ -69,6 +72,18 @@ impl PartialEq for Store {
     }
 }
 
+unsafe impl TrapHandler for Store {
+    #[inline]
+    fn as_any(&self) -> &dyn Any {
+        self
+    }
+    fn custom_trap_handler(&self, _call: &dyn Fn(&TrapHandlerFn) -> bool) -> bool {
+        // Setting a custom handler is implemented for now, please open an issue
```

Comment:
```suggestion
        // Setting a custom handler is unimplemented for now, please open an issue
```

Also, this sort of "please open an issue in github repo" is a very odd thing to do, I don't like it. I don't this needs an comment (I think my previous review asked for a comment, but I meant for the definition of the `trait TrapHandler` which already had a comment anyways?).

Reply:
Yeah, probably better to at least open the issue ourselves so discussion gets centralized and we can update people about it there.

Reply:
Just implemented it!